composer.json

```diff
@@ -26,11 +26,11 @@
     }
   },
   "require": {
-    "php": "^7.1",
+    "php": "^7.2",
```

Comment:
The minimum supported PHP version is now PHP 7.2
https://github.com/guzzle/guzzle/blob/master/UPGRADING.md